src/lxc/start.c

```diff
@@ -944,6 +944,15 @@ static int do_start(void *data)
 	if (lsm_process_label_set(NULL, handler->conf, 1, 1) < 0)
 		goto out_warn_father;
 
+	/* set sysctl value to a path under /proc/sys as determined from the key.
+	 * For e.g. net.ipv4.ip_forward translated to /proc/sys/net/ipv4/ip_forward.
+	 */
+	if (!lxc_list_empty(&handler->conf->sysctls)) {
```

Comment:
Right, so this need to move into `conf.c:lxc_setup()` right before we drop caps after setup personality so that we still have CAP_SYS_RESOURCE and friends.

Reply:
Thank you for pointing out the error. @brauner @stgraber